conf/distro/pelux.conf

```diff
@@ -9,7 +9,7 @@ require conf/distro/poky.conf
 MAINTAINER = "Gordan Markuš <gordan.markus@pelagicore.com>"
 DISTRO = "pelux"
 DISTRO_NAME = "PELUX"
-DISTRO_VERSION = "3.0"
+DISTRO_VERSION = "4.0"
```

Comment:
I feel it is too early to change this version number.

Reply:
why?

Reply:
We're not on version 4. It's one thing for SWF to point to "4.0-alpha" but as far as I know this doesn't mention anything about a preliminary/alpha release.

Reply:
of course, we're on 4.0 dev.